extension.js

```diff
@@ -71,7 +72,8 @@ function onScroll(actor, event) {
       return Clutter.EVENT_PROPAGATE;
   }
   Log.debug("opacity: " + opacity);
-  win_actor.set_opacity(Math.max(min_opacity, Math.min(opacity, 255)));
+  opacity = Math.max(min_opacity, Math.min(opacity, 255));
+  win_surface[0].opacity = opacity;
```

Comment:
does MetaSurfaceActor still have set_opacity method? It would be better to use a method than setting a field directly.

Reply:
Not sure about method, it's my first attempt with gnome extensions, i used [transparent-window-moving](https://github.com/Noobsai/transparent-window-moving/commit/05a29391994cb1a2c828813be32f9c0cafaa24f2) as fix example.

Reply:
Looks like MetaSurfaceActor is a subclass of 
ClutterActorClass: https://gitlab.gnome.org/GNOME/mutter/-/blob/master/src/compositor/meta-surface-actor.h#L23. Then MetaSurfaceActor should also have set_opacity method. Please update the code and I will later on merge it.